testbed/tests/app/test_screen.py

```diff
@@ -0,0 +1,52 @@
+from importlib import import_module
+
+import pytest
+
+from toga.platform import current_platform
+
+
+@pytest.fixture
+def screen_probe_list(app):
+    module = import_module("tests_backend.screen")
+    if current_platform == "android":
+        return [getattr(module, "ScreenProbe")(app, screen) for screen in app.screens]
+    else:
+        return [getattr(module, "ScreenProbe")(screen) for screen in app.screens]
```

Comment:
Why this discrepancy? The whole point of having an interface is that it is consistent across all platforms; if you need to pass in app, then pass it in for every platform. It doesn't matter if it's not *used* - having a consistent interface is more important.

Reply:
Modified it.